tests/test-team/helpers/auth/cognito-auth-helper.ts

```diff
@@ -251,7 +260,16 @@ export class CognitoAuthHelper {
         await CognitoAuthHelper.authContextFile.set(runId, id, {
           password,
         });
-        this.password = password;
+      },
+
+      async getPassword() {
+        const u = await CognitoAuthHelper.authContextFile.get(runId, id);
```

Comment:
nitpicky, ignore if you want but I hate single character variable names :D 
```suggestion
        const user = await CognitoAuthHelper.authContextFile.get(runId, id);
```

Reply:
I was aware of this but this method is in quite a nested scope and there are already variables with all the useful names! `user` and `userCtx ` were already taken. I figured for such a simple method it'd be alright